app/src/main/java/com/github/sdpteam15/polyevents/admin/ItemRequestManagementActivity.kt

```diff
@@ -0,0 +1,13 @@
+package com.github.sdpteam15.polyevents.admin
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import com.github.sdpteam15.polyevents.R
+
+class ItemRequestManagementActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_item_request_management)
+        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
```

Comment:
Do we need supportActionBar!!.setDisplayShowHomeEnabled(true) as well?

Reply:
It allows us to have the back button on the top left of the screen, so that we can quickly get back to the MainActivity. I thought it was a nice navigation improvement ^^

Reply:
Nice!